src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,117 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int CAPACITY_MULTIPLIER = 2;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
```

Comment:
this variable is redundant, simple use `table.length * LOAD_FACTOR` when it needed

Reply:
fixed